examples/step-63/doc/intro.dox

```diff
@@ -75,31 +109,54 @@ for each cell, and $\delta_K$ is a cell-wise constant
 stabilization parameter defined in <a
 href="https://link.springer.com/chapter/10.1007/978-3-540-34288-5_27">
 On Discontinuity-Capturing Methods for Convection-Diffusion Equations
-by Volker John and Petr Knobloch</a>. Essentially, adding in the
+by Volker John and Petr Knobloch</a>.
+
+Essentially, adding in the
 discrete strong form residual enhances the coercivity of the bilinear
 form $a(\cdot,\cdot)$ which increases the stability of the discrete
 solution. This method is commonly referred to as <i>streamline
 diffusion</i> or <i>SUPG</i> (streamline upwind/Petrov-Galerkin).
 
 
-<h2>Smoothers</h2>
+<h3>Smoothers</h3>
 
 One of the goals of this tutorial is to expand from using a simple
 (point-wise) Gauss-Seidel (SOR) smoother that is used in step-16
 (class PreconditionSOR) on each level of the multigrid hierarchy.
-Here, we consider point-wise smoothers (Jacobi and SOR) and cell-based
-smoothers (Block Jacobi and Block SOR). The cell-based smoothers can
+The term "point-wise" is traditionally used to indicate that one
+solves at one "grid point" at a time; for scalar problems, this means
+to use a solver or smoother that solves for one unknown of the linear
+system at a time, keeping all of the others fixed; one would then
+iterate over all unknowns in the problem and, once done, start over again
+from the first unknown until these "sweeps" converge. Jacobi,
+Gauss-Seidel, and SOR iterations can all be interpreted in this way.
+
+But these methods are known to converge rather slowly when used as
+solvers, and to act as smoothers of only mediocre quality when used in
```

Comment:
I wouldn't call point smoothers "mediocre". They are very performant and, in fact, all our large scale parallel computations use point smoothers. In the results section, you can see that they easily beat the cell based smoothers in runtime.